Projects/B-L072Z-LRWAN1/Applications/LoRa/AT_Slave/LoRaWAN/App/src/lora.c

```diff
@@ -244,6 +238,9 @@ static void McpsIndication(McpsIndication_t *mcpsIndication)
         break;
       default:
 
+        if ((mcpsIndication->Region == LORAMAC_REGION_EU868) || (mcpsIndication->Region == LORAMAC_REGION_RU864) || (mcpsIndication->Region == LORAMAC_REGION_CN779) || (mcpsIndication->Region == LORAMAC_REGION_EU433)) {
```

Comment:
This snipped may or may not be necessary since it's also applied later; need to check it on the field